ublox/src/lib.rs

```diff
@@ -69,6 +69,10 @@
 #[cfg(feature = "alloc")]
 extern crate alloc;
 
+#[cfg(feature = "use-serde")]
```

Comment:
The feature should be called `serde` instead of `use-serde`.

Reply:
woops, I forgot this, implied by your 1st review,
good catch